src/large_pages/node_large_page.cc

```diff
@@ -38,12 +40,11 @@
 #elif defined(__APPLE__)
 #include <mach/vm_map.h>
 #endif
-#include <unistd.h>  // readlink
+#include <unistd.h>  // getpid
 
 #include <climits>  // PATH_MAX
 #include <clocale>
 #include <csignal>
-#include <cstdio>
```

Comment:
I think you can also remove ` #include <sstream>` now?

Reply:
Done.